pkg/config/config.go

```diff
@@ -163,11 +165,13 @@ func newConfig(buildConfig []byte, rewriteDockerRepositoryName bool) (*Config, e
 	} else {
 		if applicationType == JavaLeveransepakke {
 			applicationSpec.MavenGav.Classifier = Leveransepakke
-		} else {
+		} else if applicationType == NodeJsLeveransepakke {
 			applicationSpec.MavenGav.Classifier = Webleveransepakke
+		} else {
+			applicationSpec.MavenGav.Classifier = Doozerleveransepakke
 		}
 	}
-	if applicationType == JavaLeveransepakke {
+	if applicationType == JavaLeveransepakke || applicationType == DoozerLeveranse {
```

Comment:
Under arbeidet med bygging av architect har jeg blitt usikker på om doozerleveranse skal være låst til pakkeformat.  Mulig at openshift.json kan inneholde spec for type pakke, i.e. zip, tgz etc ?